src/libraries/action-scheduler/changelog.txt

```diff
@@ -1,5 +1,103 @@
 *** Changelog ***
 
+= 3.8.2 - 2024-09-12 =
+* Add missing parameter to the `pre_as_enqueue_async_action` hook.
+* Bump minimum PHP version to 7.0.
+* Bump minimum WordPress version to 6.4.
```

Comment:
I think we try to be compatible with WP 5.8+ ?

Reply:
right, used the version which fixed the warnings (just one minor up, `3.5.4 -> 3.5.3`), so no changes to wp min req